.pre-commit-config.yaml

```diff
@@ -5,33 +5,45 @@ repos:
     - id: gitlint
   - repo: local
     hooks:
-    - id: tests
-      name: Tests
-      description: Run project tests
+    - id: pytest
+      name: Python tests
+      description: Run project's Python tests
       entry: .venv/bin/pytest --no-cov
       pass_filenames: false
       language: python
   - repo: local
     hooks:
-    - id: linter
-      name: Linter
-      description: Run project linter
+    - id: prospector
```

Comment:
prospector hook did not work (as it is not installed in the virtual environment of the project, it complains of missing imports)